### ISSUE TITLE
The `.send()` method improvements

### ISSUE THREAD

**Issue by koresar** [bbc/sqs-producer#47]
1. Looking at the code I believe that the return type of the `.send()` method is wrong.
```
async send(messages: string | string[]): Promise<string[]> {
```
The function always return undefined. I believe the return type should be `void` instead of `string[]`.

Should I submit a 1-line PR to fix this?

2. There were a number of requests from several people to return AWS raw response(s) form this function. The currently returned `undefined` is somewhat wasteful. It would be great to take the `result` variables, concat to an array, and return from the `.send()` method:

```
const result = await this.sqs.sendMessageBatch(params).promise();
results.push(result);

...

return results;
```

This is a non-breaking enhancement people really need.

Should I submit a PR?

**Comment by nspragg** [comment]
@koresar - `send` has the correct return type of `Promise<SendMessageBatchResultEntryList>`. For more details see the [source](https://github.com/bbc/sqs-producer/blob/master/src/producer.ts#L41)

This was implemented in #36 

However, the release was invalid. This has now been corrected. Thanks.

**Comment by koresar** [comment]
That's great! Thank you
